tests/artemis-test-support/src/main/java/org/apache/activemq/transport/netty/NettyWSTransport.java

```diff
@@ -80,16 +80,13 @@ public NettyWSTransport(NettyTransportListener listener, URI remoteLocation, Net
    }
 
    @Override
-   public ChannelFuture send(ByteBuf output) throws IOException {
-      checkConnected();
-      int length = output.readableBytes();
-      if (length == 0) {
-         return null;
-      }
-
-      LOG.trace("Attempted write of: {} bytes", length);
```

Comment:
Should keep trace logging in new methods for future debugging of issues in live systems

Reply:
I don't think this one is very useful...

I will merge it.